src/hooks/useDimensions.ts

```diff
@@ -1,11 +1,4 @@
 import { useWindowDimensions } from 'react-native'
 import type { ScreenSize } from '../types'
 
-export const useDimensions = (): ScreenSize => {
-    const { width, height } = useWindowDimensions()
-
-    return {
-        width,
-        height
-    }
-}
+export const useDimensions = (): ScreenSize => useWindowDimensions()
```

Comment:
Unwrapping this object and creating a new object breaks memoization. I was going to reach for another `useMemo` here, but the object is already stable direct from React Native, so we might as well consume it directly.